### ISSUE TITLE
Display discard analyses without having to scroll in mobile landscape mode

### ISSUE THREAD

**Issue by markafitzgerald1** [markafitzgerald1/cribbage-trainer#2]
Because the analyses always display below the cards at present the analyses are off-screen and require scrolling in mobile landscape mode.

**Comment by markafitzgerald1** [comment]
May need to make the size of 'vw' units fonts consistent with non-'vw' font of each calculation to provide horizontal space in which the analyses can be displayed in landscape mode.

Be sure to add an automated regression test for this via mobile Playwright tests in landscape mode.

**Comment by markafitzgerald1** [comment]
`@media` queries can be used to make the cards be less than full viewport width in landscape mode. e.g.:
```
@media screen and (orientation: landscape) {
  div {
    width: 50%;
  }
}
```

### TODO

- [ ] Sketch ideal portrait and landscape layouts before coding the above and related style and DOM ordering changes up.

**Comment by markafitzgerald1** [comment]
Sketches done. Key is to move the analysis to the right of the sort controls and hand. Title can span both - or not. To do this will need to wrap the sort+analysis in another DOM element to style it to the left.

**Comment by markafitzgerald1** [comment]
Resolved in 6a2fa4a0fbd87cc98a2fef27144515038926f479 and 84449dcb5d1b018b851c144c648b7203c1cbd4e9.
